docs/components/saas/byok/aws-kms-setup.md

```diff
@@ -0,0 +1,132 @@
+4. Under **Encryption at rest**, choose **External**.
+5. Click **Create Cluster**.
+
+:::note Screenshot placeholder
+[Insert screenshot of cluster creation page with "External" encryption selected]
+:::
+
+After creation, the Console will display a **unique Camunda Cluster IAM role ARN**. You will use this in the next step when configuring your KMS key.
+
+## Step 2: Create and configure a KMS key in AWS
+
+Follow these steps in your AWS account:
+
+1. Open the **AWS KMS console**.
+2. Click **Create key**.
+3. Choose **Symmetric** key type.
+4. Select **Region** matching your Camunda cluster.
+5. Click **Next: Key administrators** and assign at least one administrator.
+6. Click **Next: Key usage permissions**. Here, you need to grant your Camunda cluster role access.
+
+### Example KMS key policy
+
+Replace `<tenant-iam-role-arn>` with the IAM role ARN from Step 1:
+
+```json
+{
+  "Sid": "Allow Camunda tenant IAM Role basic key access",
+  "Effect": "Allow",
+  "Principal": {
+    "AWS": "<tenant-iam-role-arn>"
+  },
+  "Action": [
+    "kms:DescribeKey",
+    "kms:GenerateDataKey*",
+    "kms:Decrypt"
+  ],
+  "Resource": "*"
+},
+{
+  "Sid": "Allow Camunda tenant IAM Role to create grants for provisioning encrypted EBS volumes",
+  "Effect": "Allow",
+  "Principal": {
+    "AWS": "<tenant-iam-role-arn>"
+  },
+  "Action": [
+    "kms:CreateGrant"
+  ],
+  "Resource": "*"
+}
+```
+
+:::warning Key policy warnings
+
+- Ensure you do not restrict the Camunda cluster IAM role from any required KMS actions; otherwise encryption will fail.
+- Keys must remain in the same region as your cluster.
+- Any revocation of access will immediately break the Camunda cluster.
+
+:::
+
+7. Click **Finish** to create the key.
+8. Copy the **Key ARN**; you will need it in the Camunda Console.
+
+:::note Screenshot placeholder
+[Insert screenshot of AWS KMS key details showing Key ARN]
+:::
+
+## Step 3: Associate KMS key with Camunda cluster
+
+1. Return to the Camunda Console.
+2. In the cluster details page, locate the KMS Key ARN input field.
+3. Paste your AWS KMS Key ARN from Step 2.
+4. Confirm and apply the key. The Camunda Operator will provision storage using your BYOK key for:
+   1. Document handling storage
+   2. Backup storage
+   3. Orchestration Cluster persistent disks
+   4. Elasticsearch persistent disks
+
+:::note Screenshot placeholder
+[Insert screenshot showing KMS Key ARN input in cluster details]
+:::
+
+## Step 4: Verify encryption and logging
+
+- AWS CloudTrail automatically logs all KMS operations.
+- Set up CloudWatch alerts for KMS key deletion or unauthorized access attempts.
+- Verify your cluster’s storage volumes are using the correct KMS key. This is possible, but not required.
+
+:::warning Monitoring reminder
+You are responsible for monitoring your key usage and access logs.
+:::
+
+## Additional considerations
+
+- **Key rotation**: AWS supports manual rotation; Camunda 8.8 does not rotate customer keys.
```

Comment:
When we say manual, we refer to external to Camunda. KMS rotation is not manual, as customers have automated procedures to rotate keys.
Using the manual term here may be confusing. Can we say something inline that Amazon KMS must perform key rotation as it owns the key lifecycle?